src/Dfe.PlanTech.Domain/Questionnaire/Models/QuestionDbEntity.cs

```diff
@@ -15,9 +15,6 @@ public class QuestionDbEntity : ContentComponentDbEntity, IQuestion<AnswerDbEnti
 
     public string Slug { get; set; } = null!;
 
-    [DontCopyValue]
-    public List<AnswerDbEntity> PreviousAnswers { get; set; } = [];
```

Comment:
Just for interest, what did we used to use this for?

Reply:
Nothing :D